app/WebParsing/UtsgJsonParser.hs

```diff
@@ -47,44 +47,37 @@ insertAllCourses org = do
         -- only sections are currently stored here.
         (_, sections) = unzip courseData
         meetings = concat sections
-    mapM_ insertCourse meetings
+    mapM_ insertMeeting meetings
 
-getCourseKey :: T.Text -> SqlPersistM (Maybe (Key Courses))
-getCourseKey code = do
-    keyListCourse :: [Key Courses] <- selectKeysList [ CoursesCode ==. code ] []
-    return $ case keyListCourse of
-        [] -> Nothing
-        _ -> Just (head keyListCourse)
-
-insertCourse :: Meeting -> SqlPersistM ()
-insertCourse meet = do
-    courseKey <- getCourseKey (meetingCode meet)
+-- | Store a meeting's data and times.
+insertMeeting :: MeetTimes -> SqlPersistM ()
+insertMeeting meet = do
```

Comment:
Use [pattern-matching](https://en.wikibooks.org/wiki/Haskell/Pattern_matching#Introduction_to_records) on `meet` to destructure it here, so that you don't need to call `meetingData` and `timesData` in the function body.